### ISSUE TITLE
[Proposal] Rename `--match-edid` to `--ignore-outputname`

### ISSUE THREAD

**Issue by 9las** [phillipberndt/autorandr#401]
I propose renaming the `--match-edid` option to `--ignore-outputname` (or a similar name) to better reflect its actual behavior.

The current name, `--match-edid`, is somewhat misleading because **autorandr always matches displays by EDID**, even without this option. What the `--match-edid` option truly changes is the behavior of how output names are handled—it tells **autorandr** to **ignore output names** and rely solely on EDID for matching.

Renaming the option to something like `--ignore-outputname` would make its functionality more intuitive, as it emphasizes what is being ignored (the output name) rather than what is already a default behavior (EDID matching).

**Comment by phillipberndt** [comment]
Good point, but I won't be deprecating existing options at this point in the lifetime of this program. We could either introduce the new name and (silently?) continue to support the old, or just improve the documentation.

**Comment by 9las** [comment]
Introducing the new name and silently continue to support the old sounds like a good solution to me.
